extensions/positron-python/src/client/positron/extension.ts

```diff
@@ -102,40 +96,3 @@ export async function activatePositron(serviceContainer: IServiceContainer): Pro
         traceError('activatePositron() failed.', ex);
     }
 }
-
-export async function checkAndInstallPython(
```

Comment:
Moved this to fix a circular dependency and because its only usage is in `manager.ts`